tests/study_tests/test_study.py

```diff
@@ -799,9 +799,21 @@ def _objective(trial: Trial) -> float:
 def test_optimize_with_progbar_parallel(capsys: _pytest.capture.CaptureFixture) -> None:
 
     study = create_study()
-    with pytest.warns(UserWarning, match="Progress bar only supports serial execution"):
-        study.optimize(lambda _: 1.0, n_trials=10, show_progress_bar=True, n_jobs=-1)
+    study.optimize(lambda _: 1.0, n_trials=10, show_progress_bar=True, n_jobs=-1)
```

Comment:
This is not actually a change of this PR, but I think `n_jobs=-1` makes the test environmental dependent. How about setting `n_jobs=2` as can be seen in `test_optimize_with_progbar_parallel_timeout`?
```suggestion
    study.optimize(lambda _: 1.0, n_trials=10, show_progress_bar=True, n_jobs=2)
```

Reply:
Ok, fixed.